### PR TITLE
Add breaking change doc for cupertino icons package update

### PR DESCRIPTION
**Pull request by xster**
For the soft visual breaking change in https://github.com/flutter/flutter/pull/65083.

### CODE REVIEW

File: src/docs/release/breaking-changes/cupertino-icons-1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+At this point, you can also explore [CupertinoIcons][]
+or [flutter.github.io/cupertino_icons][] for new icons to use in your
+application.
+
+You're encouraged to verify your application after migrating to ensure that
+the automatically mapped new icons are suitable for your desired aesthetics.
+
+## Timeline
+
+Landed in: https://github.com/flutter/flutter/pull/65083<br>
```
Comment (sfshaza2) [inline src/docs/release/breaking-changes/cupertino-icons-1.0.0.md:113]: Can we translate this to something meaningful, like 1.22? I've tried and can't figure it out.

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Except for that one piece of feedback on the timeline, this LGTM.

File: src/docs/release/breaking-changes/cupertino-icons-1.0.0.md (hunk 1 above)
Comment (xster) [reply src/docs/release/breaking-changes/cupertino-icons-1.0.0.md]: done

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Pull request merged**